tasks/postgresql.yml

```diff
@@ -0,0 +1,24 @@
+---
+# PostgreSQL tasks specific to the duffy role
+
+- name: Install PostgreSQL client packages
+  package:
+    name: "{{ postgresql_client_pkgs | list }}"
+    state: present
+  tags:
+    - pkgs
+
+
+- become_user: postgres
+  block:
+    - name: Create the duffy PostgreSQL user
```

Comment:
ansible has native modules (through community.postgresql collection) that can be used.
For example, see how we did it for kojihub https://github.com/CentOS/ansible-role-kojihub/blob/master/tasks/postgresql.yml#L7 , using the `postgresql_user` and `postgresql_db` modules from ansible